sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -2134,30 +2134,145 @@ case class OctetLength(child: Expression)
  * A function that return the Levenshtein distance between the two given strings.
  */
 @ExpressionDescription(
-  usage = "_FUNC_(str1, str2) - Returns the Levenshtein distance between the two given strings.",
+  usage = """
+    _FUNC_(str1, str2) - Returns the Levenshtein distance between the two given strings.
+      If threshold is set and distance more than it, return -1.""",
```

Comment:
Please, add the parameter that you mentioned: `_FUNC_(str1, str2[, threshold])`

Reply:
ok